README.md

```diff
@@ -83,6 +83,9 @@ code name
 # 0x14 formerly had the name "sha3", now deprecated
 ```
 
+## Compatibility note
+
+Implementations should make sure keep the string code `sha3` as an alias for `sha3-512` so that `encode('sha3', digest) == encode('sha3-512', digest)`.
```

Comment:
Should be: "...should make sure _to_ keep the..."

Content sounds right, if 512 is the default bit-size of the digest. But I'm not sure where that assumption is coming from, as there are 3 other SHA-3 hashing functions, and two more if you count the XOFs.